blog/2024-11-20-decompressing-work/index.mdx

```diff
@@ -0,0 +1,211 @@
+---
+slug: from-platform-to-destination
+title: "From Platform to Destination"
```

Comment:
Title should be a little clearer - "How your Resources get from Promise to Destination" or "Explaining the mysterious Works and WorkPlacements..." or something

Reply:
"One for the observers: What Works and WorkPlacements do"

Reply:
went with How your Resources get from Promise to Destination